podcasts/en.lproj/Localizable.strings

```diff
@@ -3389,3 +3389,27 @@
 
 /* When loading the End of Year stats stories fails. */
 "eoy_stories_failed" = "Failed to load stories.";
+
+/* Title of the view displayed after a user successfully upgrades to plus */
+"welcome_plus_title" = "Thank you, now let’s get you listening!";
```

Comment:
Should it be "let’s" or "let's"? Not sure if there's any difference between those.